about.html

```diff
@@ -0,0 +1,151 @@
+    crossorigin="anonymous"></script>
+	<link rel="stylesheet" href="style.css">
+	<script defer src="script.js"></script>
+  <title>About Us</title>
+</head>
+
+<body>
+  <!-- Header & Navigation -->
+  <header>
+	<!-- header-1 -->
+    <div class="first-header bg-secondary">
+      <ul class="first-header-ul">
+        <li><a><img src="Img/facebook-letter-logo-svgrepo-com.svg" alt="Facebook"></a></li>
+        <li><a><img src="Img/twitter-white.svg" alt="twitter"></a></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+	<!-- header-2 -->
+    </div>
+    <div class="nav-div">
+      <div class="logo">
+					<a href="index.html"
+						><img class=logo src="Img/daankwaLogo.png"  alt="Logo"
+					/></a>
+				</div>
+				<ul class="nav justify-content-end">
+					<li class="navy">
+          <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link active" aria-current="page" href="about.html">About</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link" href="index.html#program">Program</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link" href="./index.html#join">Join</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link" href="./index.html#sponsors">Sponsor</a>
+        </li>
+        <li class="navy">
+          <a class="nav-link" href="#">News</a>
+        </li>
+        <li class="navy campaign">
+          <a class="nav-link" href="#">Talents</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+					<img
+						src="Img/menu-sharp-custom.svg"
+						alt="Menu"
+					/>
+				</div>
+			</div>
+		</header>
+		<!-- Content Body -->
+		<div class="main">
+			<!-- Section Headline -->
+			<section class="headline">
+				<div class="wrapper into">
+					<h1>"When it comes to forever living Daa Nkwa is the choice"</h1>
+					<h2>Corporate Reunion</h2>
+					<p class="celebration">
+						The forever living project focuses on sustainable farming practices to enhance crop production and promote environmental stewardship. The project aims to implement innovative techniques such as organic farming, precision agriculture, and water conservation methods. By utilizing organic fertilizers, minimizing pesticide use, and adopting efficient irrigation systems, the project seeks to improve soil health, optimize resource utilization, and reduce the ecological footprint of farming operations. Furthermore, the project incorporates community engagement and knowledge-sharing initiatives to empower local farmers with modern agricultural techniques and promote sustainable livelihoods. Through this project, the goal is to establish a model for sustainable agriculture that maximizes productivity while minimizing the negative impacts on the environment, leading to a more resilient and prosperous agricultural sector.
```

Comment:
- Kindly, In this part of your design, I noticed that the bottom parts with the contact information and email weren't showing in your design as seen in the Behance design. So please make sure that they show 👍

 - Also, please ensure that the contents of this section are well-centered and have similar spacing around the elements as seen in the Behance design 👍

 - Also please ensure that the background color of the paragraph text is the same as that of the Behance design. Currently, yours has a grey background color instead of white as seen in Design 💯
### Please Check the previous reviewer [comment](https://github.com/obibaadoma/captone_DaaNkwa/pull/1#discussion_r1246706392)